packages/tpetra/core/example/BlockCrs/Tpetra_TestBlockCrsMeshDatabase.hpp

```diff
@@ -314,7 +316,7 @@ namespace BlockCrsTest {
     StructuredProcGrid _grid;
     StructuredBlockPart _owned;
 
-    typedef Kokkos::View<GO*,exec_space> global_ordinal_view_type;
+    typedef Kokkos::View<GO*,mem_space> global_ordinal_view_type;
```

Comment:
What determines whether to use exec_space vs mem_space?  Why use mem_space here and not device_type as in other code changes?

Reply:
Yes, this should be device_type for consistency with the other changes. mem_space and device_type are equivalent here but it's good to be consistent.